mne/io/brainvision/brainvision.py

```diff
@@ -94,11 +94,14 @@ def __init__(self, vhdr_fname, montage=None,
         _check_update_montage(info, montage)
         with open(data_filename, 'rb') as f:
             if isinstance(fmt, dict):  # ASCII, this will be slow :(
-                n_skip = 0
-                for ii in range(int(fmt['skiplines'])):
-                    n_skip += len(f.readline())
-                offsets = np.cumsum([n_skip] + [len(line) for line in f])
-                n_samples = len(offsets) - 1
+                if self._order == 'F':  # multiplexed, channels in columns
+                    n_skip = 0
+                    for ii in range(int(fmt['skiplines'])):
+                        n_skip += len(f.readline())
+                    offsets = np.cumsum([n_skip] + [len(line) for line in f])
+                    n_samples = len(offsets) - 1
+                elif self._order == 'C':  # vectorized, channels, in rows
+                    raise NotImplementedError()
```

Comment:
This is just a sanity check because it should be caught already by `_get_vhdr_info` right?

Reply:
Yes, this is just to remind us where to implement the missing functionality.

Reply:
But now that you mention it, the `()` are not really necessary. I guess it is more Pythonic to just `raise NotImplementedError`, right?

Reply:
The line should never be hit so it doesn't matter too much.

Reply:
Yes, it doesn't matter here and actually both variants work.